mkosi.sanitizers/mkosi.conf

```diff
@@ -17,6 +17,6 @@ Environment=ASAN_OPTIONS=verify_asan_link_order=0:intercept_tls_get_addr=0
 # systemd.setenv here as there's a size limit on the kernel command line and we don't want to trigger it. We
 # don't use ManagerEnvironment= either as we want these to be set for pid1 from the earliest possible moment.
 KernelCommandLine=
-        ASAN_OPTIONS=strict_string_checks=1:detect_stack_use_after_return=1:check_initialization_order=1:strict_init_order=1:disable_coredump=0:use_madv_dontdump=1
+        ASAN_OPTIONS=strict_string_checks=1:detect_stack_use_after_return=1:check_initialization_order=1:strict_init_order=1:detect_invalid_pointer_pairs=2:handle_ioctl=1:print_cmdline=1:disable_coredump=0:use_madv_dontdump=1
         UBSAN_OPTIONS=print_stacktrace=1:print_summary=1:halt_on_error=1
```

Comment:
Should add `abort_on_error=1` here as well for ASAN_OPTIONS and UBSAN_OPTIONS no? (And in 10-sanitizers.conf as well)

Reply:
Now saniter reports are parsed from journal. So not necessary.